cmake/modules/AddSwift.cmake

```diff
@@ -981,6 +981,9 @@ function(_add_swift_library_single target name)
      "${SWIFT_SDK_${SWIFTLIB_SINGLE_SDK}_OBJECT_FORMAT}" STREQUAL "ELF")
     list(APPEND link_flags "-fuse-ld=gold")
   endif()
+  if("${SWIFTLIB_SINGLE_SDK}" STREQUAL "FREEBSD")
+    list(APPEND link_flags "-fuse-ld=gold")
```

Comment:
Hmm, where does it default to TRUE? For some reason, it doesn't on my end, even after a pull from upstream (apple/swift repo).. The error is:

--
/usr/bin/ld: stdlib/public/core/freebsd/x86_64/Swift.o: relocation R_X86_64_PC32 against protected function `_swift_stdlib_unicode_strToLower' can not be used when making a shared object
/usr/bin/ld: final link failed: Bad value
--
Which I believe was discussed on the mailing list and is happening also on Linux.